components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -521,14 +522,16 @@
                         </label>
                     </div>
                 </div>
-
                 <div style="clear:both"></div>
                 <!-- sectionSub Div -->
                 <br/>
+                <%
+                    if(FrameworkUtils.isAdaptiveAuthenticationAvailable()) {
+                %>
```

Comment:
how we are going to handle this in our console ?

Reply:
with this PR https://github.com/wso2/identity-apps/pull/3116